core/src/main/scala/kafka/utils/KafkaScheduler.scala

```diff
@@ -32,14 +32,12 @@ trait Scheduler {
   /**
    * Initialize this scheduler so it is ready to accept scheduling of tasks
    */
-  def startup()
-  
+  def startup(): Unit
   /**
    * Shutdown this scheduler. When this method is complete no more executions of background tasks will occur. 
    * This includes tasks scheduled with a delayed execution.
    */
-  def shutdown()
-  
```

Comment:
Is there a reason for the space removal here? I would think it reads better when there is some space in between

Reply:
That was a minor mistake on my part. Fixing.